rest/v9/channel.ts

```diff
@@ -130,6 +130,12 @@ export interface RESTPatchAPIChannelJSONBody {
 	 * Channel types: text, news
 	 */
 	default_auto_archive_duration?: ThreadAutoArchiveDuration;
+	/**
+	 * Whether non-moderators can add other non-moderators to the thread
+	 *
+	 * Channel types: privateThread
```

Comment:
What's this?

Reply:
?

Reply:
What does it represent? Can you be more specific wkth the jsdoc, like `Only present on the following channel typez: {@link ChannelTypes.PrivateThread}` or similar

Reply:
well all the other fields use the same format, should I change all of them? that feels out of the scope of this PR

Reply:
Do they..? Ugh.. Yeah, I guess for consistency, leave it as is for now, but we gotta fix that